xbmc/cores/VideoPlayer/DVDSubtitles/webvtt/WebVTTHandler.cpp

```diff
@@ -248,11 +251,13 @@ void CWebVTTHandler::Reset()
   m_offset = 0;
 }
 
-bool CWebVTTHandler::CheckSignature(const std::string& data)
+bool CWebVTTHandler::CheckSignature(const std::string& signature_line)
```

Comment:
variable name not conform, please give a fast read to our code guidelines

```suggestion
bool CWebVTTHandler::CheckSignature(const std::string& signatureLine)
```

Reply:
Snake-case crept in from muscle memory...

Will update.